docs/resources/supported_operations.md

```diff
@@ -0,0 +1,1316 @@
+       ShuffleChannels                    Supported**      Supported       Not Supported     Supported       
+       Sign                               Supported**      Supported       Not Supported     Supported       
+       Sin                                Supported**      Supported       Not Supported     Supported       
+       Sinh                               Supported**      Supported       Not Supported     Supported****
+       SimplerNMS                         Supported**      Supported       Not Supported     Not Supported   
+       Slice                              Supported***     Supported       Supported         Not Supported   
+       SoftMax                            Supported***     Supported       Not Supported     Supported       
+       Softplus                           Supported**      Supported       Not Supported     Supported       
+       Softsign                           Supported**      Supported       Supported         Not Supported   
+       SpaceToDepth                       Supported**      Not Supported   Not Supported     Supported*     
+       SpatialTransformer                 Supported**      Not Supported   Not Supported     Not Supported   
+       Split                              Supported***     Supported       Supported         Supported       
+       Squeeze                            Supported**      Supported       Supported         Supported       
+       StridedSlice                       Supported**      Supported       Not Supported     Supported*     
+       Tan                                Supported**      Supported       Not Supported     Supported****
+       TensorIterator                     Supported        Not Supported   Supported         Supported       
+       Tile                               Supported***     Supported**     Not Supported     Supported       
+       TopK                               Supported**      Supported       Not Supported     Supported****
+       Unpooling                          Not Supported    Supported       Not Supported     Not Supported   
+       Unsqueeze                          Supported**      Supported       Supported         Supported       
+       Upsampling                         Not Supported    Supported       Not Supported     Not Supported   
+      =================================  ===============  ==============  ================  ==================
+      
+      |   `*` - support is limited to the specific parameters. Refer to "Known Layer Limitations" section for the device :doc:`from the list of supported <openvino_docs_OV_UG_supported_plugins_Supported_Devices>`.
+      |   `**` - support is implemented via :doc:`Extensibility mechanism <openvino_docs_Extensibility_UG_Intro>`.
+      |   `***` - supports NCDHW layout.
+      |   `****` - support is implemented via runtime reference.
+
+
+
+
+
+   .. tab-item:: Operations Supported by Other Frameworks
```

Comment:
It is not correct to call it as `Operations Supported by Other Frameworks`.
Please separate into two documents: `Operations supported by OpenVINO plugins` and `Operations supported by OpenVINO Framework Frontend`